### PR TITLE
Replace "Return" with "Returns" in API documentations

### PR DESCRIPTION
**Pull request by sato11**
### Motivation / Background
The guidelines are explicit about how we should start sentences with "Returns" rather than "Return." So these instances should follow that way.

### Additional information
See also the enhancements made at https://github.com/rails/rails/pull/50595.

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Changes that are unrelated should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

File: activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb [hunk 1]
```diff
@@ -475,7 +475,7 @@ def num_waiting_in_queue # :nodoc:
         @available.num_waiting
       end
 
-      # Return connection pool's usage statistic
+      # Returns the connection pool's usage statistic
       # Example:
```
Comment (jonathanhefner) [inline activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb:479]: This one is public. :+1:  We can tidy it up a bit though:

```suggestion
      # Returns the connection pool's usage statistic.
```

File: activerecord/lib/active_record/connection_handling.rb [hunk 2]
```diff
@@ -256,7 +256,7 @@ def connection
 
     attr_writer :connection_specification_name
 
-    # Return the connection specification name from the current class or its parent.
+    # Returns the connection specification name from the current class or its parent.
```
Comment (jonathanhefner) [inline activerecord/lib/active_record/connection_handling.rb:259]: This one is public. :+1:

**Review by jonathanhefner — COMMENTED** [review]
Most of these comments are private, so I don't think they need to be changed.  I commented on the two public ones though.

**Comment by sato11** [comment]
Thanks for the comments, @jonathanhefner!

To confirm, you want me to discard the changes made to the private items, right? Let me confess that I'm not able to find it problematic to change all. So if you have more details behind your words, I'd like to hear it 🙂 

That said, I'm fine to limit the change only to the public ones 👍 So I'll work on it for now.

**Comment by jonathanhefner** [comment]
> To confirm, you want me to discard the changes made to the private items, right? Let me confess that I'm not able to find it problematic to change all. So if you have more details behind your words, I'd like to hear it

Correct.  Private comments are mostly exempt from the documentation guidelines because they are not public facing.  So, for private comments, "Return" => "Returns" would be a cosmetic change, which we generally avoid.  Another reason is to avoid noise in `git blame` when there isn't much benefit.

Thank you, @sato11! :leftwards_arrow_with_hook:

**Comment by sato11** [comment]
Thanks for pointing it out @jonathanhefner, and thanks again for the review which must have been tedious 😢 I'll remember it 👍

**Pull request merged**